apps/frontend/src/pages/not-found/styles.module.css

```diff
@@ -0,0 +1,47 @@
+.container {
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	color: var(--color-text-primary);
+	background-color: var(--color-background-primary);
+}
+
+.title {
+	margin: 0;
+	font-size: 96px;
+	font-weight: 600;
+	line-height: 116.18px;
```

Comment:
I was thinking this was the main title 😅 in case it's just a decoration, we don't need line-height here. Also for the future, it's ok to round values a bit to make them whole numbers at least.
```suggestion
```